File: src/main/java/com/yauhenl/neuro/Environment.java
package com.yauhenl.neuro;

import processing.core.PApplet;

public class Environment extends PApplet {

    private com.yauhenl.neuro.World world;

    @Override
    public void setup() {
        world = new com.yauhenl.neuro.World(this, 15, 50);
    }

    @Override
    public void settings() {
        size(1000, 600);
    }

    @Override
    public void draw() {
        background(33);
        world.draw();
    }

    public static void main(String[] args) {
        PApplet.main(Environment.class);
    }
}

File: src/main/kotlin/com/yauhenl/kotlin/neuro/Environment.kt
package com.yauhenl.kotlin.neuro

import processing.core.PApplet

class Environment : PApplet() {

    private var world: World? = null

    override fun setup() {
        world = World(this, 15, 50)
    }

    override fun settings() {
        size(1000, 600)
    }

    override fun draw() {
        background(33)
        world?.draw()
    }
}

fun main(args: Array<String>) {
    PApplet.main(Environment::class.java)
}